tests/test_add_images.py

```diff
@@ -0,0 +1,42 @@
+import clesperanto as cle
+
+
+import numpy as np
+
+input1 = np.asarray([[1, 2, 3]])
+input2 = np.asarray([[4, 5, 6]])
+
+
+def test_add_images_weighted_missing_params():
+    reference = np.asarray([[5, 7, 9]])
+    output = cle.add_images_weighted(input1, input2)
+    result = cle.pull(output)
+    assert np.array_equal(result, reference)
```

Comment:
You might want to use [np.testing.assert_equal](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_equal.html#numpy.testing.assert_equal) here and below instead?

Reply:
agreed.  I started changing everything to the `np.testing` module in `tests/test_cle.py` ... and so far was just "pytestifying" the code in `test_add_images` before cleaning it up

Reply:
May I ask why? Curious

Reply:
it's honestly not _that_ big of a difference, and there are pros/cons and differing opinions on each side.  See, for example, this thread: https://github.com/scipy/scipy/issues/8583
and discussion here: https://numpy.org/doc/stable/reference/testing.html
I have a slight preference for np.testing ... for point # 2 in that first link above:

> Guaranteed support. numpy.testing can be guaranteed to work as long as numpy does, whereas assert relies on pytest maintenance and/or pytest-compatible assert support by any future testing framework.